db/schema.sql

```diff
@@ -172,7 +172,7 @@ CREATE TYPE COLLECT_JOB_STATE AS ENUM(
 -- The leader's view of collect requests from the Collector.
 CREATE TABLE collect_jobs(
```

Comment:
Since the standard now refers to this concept as a "collection job" rather than a "collect job", I think we should make the corresponding rename in our codebase. That is, I think this table should be renamed to "collection_jobs" & all variables named something like "collect_job_id" should be renamed to something like "collection_job_id".

(This sounds like a lot of work, and it will impact a bunch of lines, but I think it's the equivalent of doing `s/collect_job/collection_job/g` across the codebase.)

Reply:
I gave this a try, let's see what breaks! I think changing the container names might cause some minor havoc.